scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
+
+However, the SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a
+minimum. On a normal ethernet network, this should be set to around 1450. On a
+jumbo frame ethernet network, this should be around 8950.
+
+To configure the MTU, edit the node configuration file at *_/etc/origin/node/node-config.yaml_* and edit the following:
```

Comment:
I would assume that's not strictly enough -- don't we have to restart the kubelet as well?  What about any running pods, don't they have to be restarted? 

I am worried we actually need some helper plays in openshift-ansible to make toggling the MTU safe from a supportability/UX standpoint.